### ISSUE TITLE
What **query** means in the paper?

### ISSUE THREAD

**Issue by VoiceBeer** [facebookresearch/DomainBed#117]
Hi, thanks for your brilliant work! The questions raised are thought-provoking, and the standards help the game be rigorous!

In Section 3.1, I'm wondering what query means in **oracle** model selection method? To me, one query seems like a random search to find the fit hyperparameters, but do you make queries in the other two model selection methods?

Many thanks to your work and have a good day :>

**Comment by lopezpaz** [comment]
One query = one hparam configuration that we try. Oracle is only different in the way it chooses the best model (looking at validation set drawn from the test domain).

**Comment by VoiceBeer** [comment]
Hi David@lopezpaz , thx for your reply! Have a good day xD
